go_password_generator_cli/cmd/root.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var passgen = &cobra.Command{
+	Use:   "passgen",
+	Short: "Generate a random password",
+	Long:  `Passgen is a tool that generates a random password`,
+	Run: func(cmd *cobra.Command, args []string) {
+		password := generatePassword()
+		fmt.Println(password)
+	},
+}
+
+func generatePassword() string {
+	length := rand.Intn(7) + 10
```

Comment:
can you change this logic please.
Add an extra flag for the cli to specify if the user wants 10 or 16 characters. If nothing specified it should be of length 10 else if something is provided it should be 16 characters.
You can keep the flag as a boolean.

Reply:
ok, working on it.